src/garage/tf/misc/tensor_utils.py

```diff
@@ -52,19 +53,21 @@ def flatten_batch(t, name='flatten_batch'):
 
 
 def flatten_batch_dict(d, name=None):
-    with tf.name_scope(name, 'flatten_batch_dict', [d]):
+    with tf.name_scope('flatten_batch_dict' if name is None else name):
         return {k: flatten_batch(v) for k, v in d.items()}
 
 
 def filter_valids(t, valid, name='filter_valids'):
     # 'valid' is either 0 or 1 with dtype of tf.float32
     # Must round before cast to prevent floating-error
-    return tf.dynamic_partition(
-        t, tf.cast(tf.round(valid), tf.int32), 2, name=name)[1]
+    return tf.dynamic_partition(t,
+                                tf.cast(tf.round(valid), tf.int32),
+                                2,
+                                name=name)[1]
 
 
 def filter_valids_dict(d, valid, name=None):
-    with tf.name_scope(name, 'filter_valids_dict', [d, valid]):
+    with tf.name_scope('filter_valids_dict' if name is None else name):
```

Comment:
aren't these all equivalent to `name='filter_valids_dict'` etc.?

Reply:
I thought someone may give it a special name scope, i.e. name is not None

Reply:
after checking, there's no usage that pass a name to these functions. I will remove the name argument.

Reply:
e.g. in NPO
```
...
                policy_state_info_vars_flat = flatten_batch_dict(
                    policy_state_info_vars, name='policy_state_info_vars_flat')
                policy_old_dist_info_vars_flat = flatten_batch_dict(
                    policy_old_dist_info_vars,
                    name='policy_old_dist_info_vars_flat')
```

Reply:
...but why not just use the default function argument to achieve that?

Reply:
I see, I missed your point